moveit_experimental/moveit_jog_arm/config/ur_simulated_config.yaml

```diff
@@ -7,9 +7,15 @@ use_gazebo: true # Whether the robot is started in a Gazebo simulation environme
 robot_link_command_frame:  world  # commands must be given in the frame of a robot link. Usually either the base or end effector
 command_in_type: "speed_units" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
 scale:
-  linear:  0.6  # Max linear velocity. Meters per publish_period. Units is [m/s]. Only used if command_in_type=="unitless"
-  rotational:  0.3 # Max angular velocity. Rads per publish_period. Units is [rad/s]. Only used if command_in_type=="unitless"
-  joint: 0.3  # Max joint angular/linear velocity. Rads or Meters per publish period. Units is [rad/s] or [m/s].
+  # Scale parameters are only used if command_in_type=="unitless"
+  linear:  0.6  # Max linear velocity. Meters per publish_period. Unit is [m/s]. Only used for Cartesian commands.
+  rotational:  0.3 # Max angular velocity. Rads per publish_period. Unit is [rad/s]. Only used for Cartesian commands.
+  # Max joint angular/linear velocity. Rads or Meters per publish period. Only used for joint commands on joint_command_in_topic.
+  joint: 10
+# Joint velocity limit on outgoing command to robot joints.
+# Unit is rad/s. This velocity limit is checked before the command is published from jog_arm.
+# Note: joints likely have additional velocity limits in the SRDF that are also checked.
+max_joint_velocity: 1
```

Comment:
So this is different than ``max_velocity_scaling_factor`` that we use elsewhere in MoveIt?

And different than the ``joint_limits.yaml`` file we use elsewhere in MoveIt (via SRDF, as you mentioned)?

Does this mean that no joint can exceed this one value? What if some joints are revolute and others prismatic? This single variable seems short-sighted.

Reply:
@henningkayser thoughts?

Reply:
@henningkayser thoughts?

Reply:
I would actually be happy to drop this parameter and just check the SRDF joint velocity limits.

I want to keep some of the other functionality from this PR, though, like the updated integration test.

Reply:
I guess the only argument for keeping that parameter in this PR is if the user wanted to launch jogging for a very slow, delicate motion (or something like that) without modifying the robot SRDF.

Reply:
@davetcoleman I totally relate to your confusion (see comment further below) ;)

> I guess the only argument for keeping that parameter in this PR is if the user wanted to launch jogging for a very slow, delicate motion (or something like that) without modifying the robot SRDF.

I see.. how about JogArm just looks for max_acceleration_scaling_factor and max_velocity_scaling_factor parameters?